manager/pom.xml

```diff
@@ -158,6 +220,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${surefire-plugin.version}</version>
+                <configuration>
```

Comment:
`maven-surefire-plugin` is configured in line 198. You may want to move this configure there :+1:

Reply:
Thanks, I missed that when I rebased :slightly_smiling_face: